src/components/Switch/index.tsx

```diff
@@ -287,6 +288,7 @@ const Switch = ({
           </SwitchBallContainer>
           <StyledCheckbox
             id={id || name}
+            aria-label={`${name}-label`}
```

Comment:
accessibility wise this is worst than having nothing, we are ticking a11y checks with placeholder data

Reply:
Could we force either aria-label or label instead ?

Reply:
Label needs real content, we don't always have something to write as a label. Maybe we should add a new `label` prop and set this `aria-label={label}`? cc @matthprost

Reply:
@QuiiBz Is there an issue of using name and put it into `aria-label={name}` ? If there is then yes I agree you can create a prop `label` and set `aria-label={label}`.